Automattic/woocommerce-payments
# Fix the "Learn More" link on Express Checkout settings section

Fixes #5083 

#### Changes proposed in this Pull Request

This minor PR fixes the `Learn More` link on the Express Checkout settings 

![image](https://user-images.githubusercontent.com/4162931/201315655-099132c9-5578-4b22-8bf4-3b4ed337f766.png)


#### Testing instructions

* Browse to `WP Admin > Payments > Settings` , and click on `Learn more` below the Express checkout section, as seen in the screenshot above. 
* The link should lead to https://woocommerce.com/document/payments/settings-guide/#section-4

-------------------

- [x] Covered with tests (or have a good reason not to test in description ☝️) - minor change. 
- [x] Tested on mobile (or does not apply) - Not needed

**Post merge**
